students/csdotson/lesson04/dict_lab.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+### Dictionaries 1 ###
+print("*" * 5 + " Dictionaries 1 " + "*" * 5)
+d = {'name': 'Chris', 'city': 'Seattle', 'cake': 'Chocolate'}
+print(d)
+
+d.pop('cake')
+print(d)
+
+d['fruit'] = 'Mango'
+print(d)
+
+print(d.keys())
+print(d.values())
+
+print("Is 'cake' a key in the dictionary?", 'cake' in d)
+print("Is 'Mango' a value in the dictionary?", 'Mango' in d.values())
+print()
+
+### Dictionaries 2 ###
+print("*" * 5 + " Dictionaries 2 " + "*" * 5)
+print("Dictionary from part 1:", d)
+for key in d.keys():
```

Comment:
note that you don't have to call `keys()`. `for key in d:` will iterate over keys